src/jobs/BenchmarkJob.jl

```diff
@@ -236,12 +236,12 @@ function Base.run(job::BenchmarkJob)
 
     # as long as our primary job didn't error, run the comparison job (or if it's a daily job, gather results to compare against)
     if !haskey(results, "error")
-        if job.isdaily # get results from previous day (if it exists, check the past 30 days)
+        if job.isdaily # get results from previous day (if it exists, check the past 120 days)
```

Comment:
Out of curiosity, what's the motivation for the bump here? 

If the daily job is regularly missing this many days since the last run, I guess it's not much of a "daily" job, huh 😁

Reply:
been a few times where the bot was down for a month, so it was better to get a comparison, rather than none at all, for those times